tegra/tegra.c

```diff
@@ -212,13 +212,16 @@ int drm_tegra_bo_wrap(struct drm_tegra_bo **bop, struct drm_tegra *drm,
 
 	/* check handle table to see if BO is already open */
 	bo = lookup_bo(drm->handle_table, handle);
+
+	pthread_mutex_unlock(&table_lock);
```

Comment:
Yeah, also I just realized that this not correct. Lookup and insert should happen under the same lock, will fix it.